rudder-core/src/main/scala/com/normation/rudder/services/servers/NewNodeManager.scala

```diff
@@ -458,6 +458,7 @@ trait ComposedNewNodeManager extends NewNodeManager with Loggable {
       case Full(seq) => //ok, cool
         logger.info(s"New node accepted and managed by Rudder: ${id.value}")
         cacheToClear.foreach { _.clearCache }
+        updateDynamicGroups.startManualUpdate
```

Comment:
Maybe cache building should trigger an update of groups ! But great ! Thanks ncharles